pkg/dns/helpers.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/api/networking/v1"
+	v1 "k8s.io/api/networking/v1"
```

Comment:
weird change. can you eliminate it?

Reply:
For some reason, vscode adds this automatically on save. Removed it with vim for now :D